spec/lib/sarif/yarn_audit_sarif_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../../spec_helper.rb'
+require 'json'
+
+describe Sarif::YarnAuditSarif do
+  describe '#parse_issue' do
+    let(:scanner) { Salus::Scanners::YarnAudit.new(repository: repo, config: {}) }
+    before { scanner.run }
+
+    context 'scan report with logged vulnerabilites' do
+      let(:repo) { Salus::Repo.new('spec/fixtures/yarn_audit/failure-2') }
+      it 'parses information correctly' do
+        issue = scanner.report.to_h.fetch(:logs).dump.split("\\n\\n")[0]
+        yarn_sarif = Sarif::YarnAuditSarif.new(scanner.report)
+
+        expect(yarn_sarif.parse_issue(issue)).to include(
+          id: "YARN0039",
+          name: "Incorrect Handling of Non-Boolean Comparisons During Minification",
+          level: "LOW",
```

Comment:
Be sure to validate any produced SARIF against the JSON schema.  Level here looks suspect.

Reply:
the testing against the SARIF schema was [moved into the sarif component](https://github.com/coinbase/salus/blob/master/lib/sarif/sarif_report.rb#L35). When we generate a SARIF report, we first check if its valid before returning it. Should i proceed to still create tests for validation?